app/(chat)/api/chat/route.ts

```diff
@@ -130,7 +130,8 @@ export async function POST(request: Request) {
         sendReasoning: true,
       });
     },
-    onError: () => {
+    onError: (error) => {
+      console.log(error);
```

Comment:
possibly rm?

Reply:
haha jinx, I just removed it as you commented!